auto_round/special_model_handler.py

```diff
@@ -45,6 +111,28 @@ def _handle_special_model(model):
     return model
 
 
+def _handle_moe_model(model):
+    if model.config.model_type in CONVERT_EXPERT_TO_LINEAR_MODELS:
+        from tqdm import tqdm
+
+        from auto_round.utils import clear_memory
+
+        new_moe_class, convert_config, orig_cls_name = _get_moe_converter(model.config)
+        model = model.to("cpu")
+        clear_memory()
+
+        for name, module in tqdm(model.named_modules(), desc="Converting model"):
+            cls_name = module.__class__.__name__
+            if cls_name == orig_cls_name:
+                new_module = new_moe_class(config=convert_config, original=module)
+                parent, child = name.rsplit(".", maxsplit=1)
+                parent = model.get_submodule(parent)
+                setattr(parent, child, new_module)
+
+        print("warning, Llama4 experts are converted, the quantized model can not run on transformers.")
```

Comment:
use logger

Reply:
special_model_handler.py is imported in utils.py, the logger is defined in utils.py, it will cuase circular import error if we import logger  from utils.py in special_model_handler.py...

Reply:
ok， then leave it for now, we will extract the logger to a sperate file later